acra/security-controls/tokenisation/_index.md

```diff
@@ -2,3 +2,170 @@
 title: Tokenisation
 bookCollapseSection: true
 ---
+
+# Tokenization
+
+## Description
+
+Tokenization is a way to transform some sensitive data into pseudo-random representation of same data type.
+A number will remain a number, string will be string, email will be email.
+But the tokenized version will look like a random value.
+Better see than read:
+
+* `789` -> `156749362`
+* `testing testing` -> `lwcIvM3zaJN9cbN`
+* `kate@example.com` -> `sVAOfOwF@mSuGPog.et`
+
+The tokenization process has few interesting properties:
+
+* Tokenized value is simply random, it is neither encrypted nor hashed version of the input
+* Tokenization of the same value more than once will return same result
```

Comment:
may return the same result (in consistent mode) or random (not consistent) even for the same value according to configuration